spec/spec_helper_acceptance_local.rb

```diff
@@ -0,0 +1,67 @@
+require 'puppet_litmus'
+require 'singleton'
+
+class Helper
+  include Singleton
+  include PuppetLitmus
+end
+
+def install_pwsh
+  script_folder = File.expand_path(File.join(File.dirname(__FILE__), 'fixtures/scripts/install_pwsh'))
+  script_path = case os[:family]
+                when 'ubuntu'
+                  File.join(script_folder, "ubuntu_#{os[:release].to_f}.sh")
+                when 'redhat'
+                  File.join(script_folder, "rhel.sh")
+                when 'debian'
+                  File.join(script_folder, "debian_#{os[:release].to_i}.sh")
+                when 'darwin'
+                  File.join(script_folder, "darwin.sh")
+                when 'windows'
+                  File.join(script_folder, "windows.ps1")
+                end
+  Helper.instance.bolt_run_script(script_path)
+end
+
+def uninstall_pwsh
+  command = case os[:family]
+            when 'ubuntu', 'debian'
+              'apt-get remove powershell -y'
+            when 'redhat'
+              'yum remove powershell -y'
+            when 'darwin'
+              'brew cask uninstall powershell'
+            when 'windows'
+              interpolate_powershell('Get-Command pwsh | ForEach-Object { Remove-Item -Path (Split-Path -Parent $_.Path) -Recurse -Force }')
+            end
+  Helper.instance.run_shell(command)
+end
+
+def pwsh_installed?
+  Helper.instance.run_shell('pwsh -v', expect_failures: true).exit_code == 0
+end
+
+def cleanup_files
+  absent_files_manifest = if os[:family] == 'windows'
+                            'file{["c:/services.txt","c:/process.txt","c:/try_success.txt","c:/catch_shouldntexist.txt","c:/try_shouldntexist.txt","c:/catch_success.txt"]: ensure => absent }'
+                          else
+                            'file{["/services.txt","/process.txt","/try_success.txt","/catch_shouldntexist.txt","/try_shouldntexist.txt","/catch_success.txt"]: ensure => absent }'
```

Comment:
Thought I'd commented on this in my previous review, but the comment isn't there...odd. 

All these files bar `/process.txt` are outputting to `/tmp` now - we may want to clean them up explicitly. I know `/tmp` will get cleared eventually, but I reckon these files will stick around if the VM remained powered on and a second test run was kicked off.

Reply:
This is a good catch, these paths needed to be updated - this method should do exactly what you're describing: ensure the files are removed prior to the test suite execution.